packages/core/src/config.ts

```diff
@@ -1220,9 +1219,18 @@ export async function initRsbuild(
     ] = config;
   }
 
-  return createRsbuild({
-    rsbuildConfig: {
-      environments,
-    },
-  });
+  return environments;
```

Comment:
Why not `composeRsbuildEnvironments` here instead of in executors of every commands

Reply:
I don't understand your statement, more detail please?

Reply:
nothing, my mistake.